tests/app/models/test_base_model.py

```diff
@@ -21,7 +21,10 @@ def foo(self):
     with pytest.raises(AttributeError) as e:
         Custom({"foo": "NOPE"})
 
-    assert str(e.value) == "can't set attribute"
+    assert (
+        str(e.value)
+        == "property 'foo' of 'test_raises_when_overriding_custom_properties.<locals>.Custom' object has no setter"
+    )
```

Comment:
Eww, the bad side of the new descriptive errors for Python shows its ugly head.

Reply:
(nothing to change here, just disturbing how much extra text needs to be verified now)

Reply:
yes, utils have more asserts like this (more descriptive messaging in 3.12)